__tests__/html/accessibility.liveRegionActivity.text.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <link href="/assets/index.css" rel="stylesheet" type="text/css" />
+    <script crossorigin="anonymous" src="/test-harness.js"></script>
+    <script crossorigin="anonymous" src="/test-page-object.js"></script>
+    <script crossorigin="anonymous" src="/__dist__/webchat-es5.js"></script>
+  </head>
+  <body>
+    <div id="webchat"></div>
+    <script>
+      function allTextContents(element) {
```

Comment:
consider refactoring into a util?

Reply:
If you do, be sure to add util description in the comments

Reply:
Refactored.